src/js/AppStoreMenu.js

```diff
@@ -0,0 +1,146 @@
+        });
+    }
+
+    confirm() {
+        fileSystemController.subscribeToEvent('UninstallApp', (success, params) => {
+            if (!success || !params.policyAppID) {
+                console.error('error encountered when uninstalling app');
+                return;
+            }
+
+            store.dispatch(addAppPendingSetAppProperties({'policyAppID': params.policyAppID}, false));
+            BCController.setAppProperties({ 
+                policyAppID: params.policyAppID,
+                enabled: false
+            });
+        });
+
+        fileSystemController.sendJSONMessage({
+            method: 'UninstallApp',
+            params: {
+                policyAppID: this.state.confirmID
+            }
+        });
+
+        this.setState((state, props) => {
+            state.confirmID = null;
+            return state;
+        });
+    }
+
+    render() {
+        const postFixes = [ "KB", "MB", "GB" ]
+        const postFixSizes = [ 1024, 1048576, 1073741824 ]
+
+        const themeClass = this.props.theme ? 'dark-theme' : 'light-theme';
+        var modalClass = themeClass + " alertOverlay"
+
+        return (
+            <div>
+                <Modal
+                isOpen={!!this.state.confirmID}
+                className="alertModal app-body"
+                overlayClassName={modalClass}
+                contentLabel="Example Modal">
+                    <ConfirmAlert appID={this.state.confirmID} name={`Uninstall ${this.state.appName}?`}
+                        description={this.state.appDescription} iconUrl={this.state.appIcon}
+                        leftText="Cancel" leftCallback={this.cancel} rightText="Uninstall" rightCallback={this.confirm} />
+                </Modal>
+                <AppHeader icon='false' backLink="/appstore" menuName="STORE"/>
+                <div className="appstore-menu">
+                    {this.props.apps ? this.props.apps.map((app) => {
+
+                        var postFixIndex = 0;
+                        while (app.package.size_decompressed_bytes > postFixSizes[postFixIndex + 1]) {
+                            postFixIndex++;
+                        }
+
+                        var size = app.package.size_decompressed_bytes / postFixSizes[postFixIndex];
+
+                        return <AppStoreMenuItem key={app.policyAppID}
+                            onSelect={() => this.onSelection(app.policyAppID, app.name, app.iconUrl, app.description)}
+                            appIcon={app.iconUrl}
+                            appName={app.name}
+                            appSize={`${size.toFixed(1)} ${postFixes[postFixIndex]}`}
+                        />
+                    }) : null}
+                </div>
+            </div>
+        )
+    }
+}
+
+const mapStateToProps = (state) => {
+    return {
+        theme: state.theme,
+        apps: state.appStore.installedApps
+    }
+}
+
+export default connect(
+    mapStateToProps
+)(AppStoreMenu)
```

Comment:
New line

Reply:
added in https://github.com/smartdevicelink/generic_hmi/pull/240/commits/b79ab73fcfd760a30437c25d53d1c6665ee29526